srcpkgs/pypy3.5/template

```diff
@@ -0,0 +1,30 @@
+# Template file for 'pypy3.5'
+pkgname=pypy3.5
+version=7.0.0
+revision=1
+wrksrc="$pkgname-v$version-src"
+makedepends="bzip2-devel expat-devel gdbm-devel libffi-devel liblzma-devel libressl-devel ncurses-devel pkg-config python python-cffi sqlite-devel zlib-devel"
```

Comment:
put `python` and `pkg-config` into `hostmakedepends`. Also add a newline after ~70 chars.

Reply:
Fixed. The commit shouldn't really be `ci skip`, but it's just so to not clog up the CI unless this is by some miracle deemed acceptable without further changes.